patches/chromium/desktop_media_list.patch

```diff
@@ -82,19 +82,10 @@ index 786c526588d81b8b5b1b5dd3760719a53e005995..f66b7d0b4dfcbb8ed3dde5a9ff463ae2
    const Source& GetSource(int index) const override;
    DesktopMediaList::Type GetMediaListType() const override;
 diff --git a/chrome/browser/media/webrtc/native_desktop_media_list.cc b/chrome/browser/media/webrtc/native_desktop_media_list.cc
-index 0827764fdeb59d339f304a20e53b01bb158104ee..3192a5b087bf88c57b77440455ae8d62edd58137 100644
+index b1d9cc5b8897b0b6d153b0b4813c8a6eca42664e..6e06fd49fb3fb1baaab15feaeac3ab93f1883508 100644
 --- a/chrome/browser/media/webrtc/native_desktop_media_list.cc
 +++ b/chrome/browser/media/webrtc/native_desktop_media_list.cc
-@@ -176,7 +176,7 @@ BOOL CALLBACK AllHwndCollector(HWND hwnd, LPARAM param) {
- #if BUILDFLAG(IS_MAC)
- BASE_FEATURE(kWindowCaptureMacV2,
-              "WindowCaptureMacV2",
--             base::FEATURE_ENABLED_BY_DEFAULT);
-+             base::FEATURE_DISABLED_BY_DEFAULT);
```

Comment:
Failing tests are related to the feature, we attempted to enable the feature https://github.com/electron/electron/pull/37014/files but haven't fixed it yet in Electron. I will keep it disabled to unblock the roll, tracking issue https://github.com/electron/electron/issues/30682